CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [v0.28.0](https://github.com/56quarters/cadence/tree/0.28.0) - 2022-01-23
```

Comment:
nit: Can you update the date here to `Unreleased`?

Reply:
Addressed in ef885a1!